src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -40,6 +40,9 @@
     <PackageReference Include="NETStandard.Library">
       <Version>$(NETStandardLibraryPackageVersion)</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.TargetingPack.Private.WinRT">
```

Comment:
It is not open source, it's a binary dependency.  The same is true in CoreFx today and has been that way since the start, but it does happen to be an on-demand reference.  It may have been true that prior to @luqunl's corefx changes the package may have never been used on linux.  Now we are building a ref and a not-supported assembly, both of which need this for public API.

I suppose if we wanted to solve that we could spike out a partial winmd using either IL or C#, similar to what we do for other reference assemblies.  @weshaggard might know better if we need to do so.

Reply:
For now we should stick with the package we have and hopefully we can exclude this on linux builds  but if not we may have to either live with the binary reference or stub it as @ericstj suggests but lets now jump there yet.